test/src/main/java/org/apache/accumulo/test/ImportExportIT.java

```diff
@@ -197,6 +204,148 @@ public void testExportImportThenScan() throws Exception {
     }
   }
 
+  @Test
+  public void testExportImportOffline() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+
+      String[] tableNames = getUniqueNames(2);
+      String srcTable = tableNames[0], destTable = tableNames[1];
+      client.tableOperations().create(srcTable);
+
+      try (BatchWriter bw = client.createBatchWriter(srcTable)) {
+        for (int row = 0; row < 1000; row++) {
+          Mutation m = new Mutation(Integer.toString(row));
+          for (int col = 0; col < 100; col++) {
+            m.put(Integer.toString(col), "", Integer.toString(col * 2));
+          }
+          bw.addMutation(m);
+        }
+      }
+
+      client.tableOperations().compact(srcTable, new CompactionConfig());
+
+      // Make export and import directories
+      FileSystem fs = cluster.getFileSystem();
+      log.info("Using FileSystem: " + fs);
+      Path baseDir = new Path(cluster.getTemporaryPath(), getClass().getName());
+      fs.deleteOnExit(baseDir);
+      if (fs.exists(baseDir)) {
+        log.info("{} exists on filesystem, deleting", baseDir);
+        assertTrue(fs.delete(baseDir, true), "Failed to deleted " + baseDir);
+      }
+      log.info("Creating {}", baseDir);
+      assertTrue(fs.mkdirs(baseDir), "Failed to create " + baseDir);
+      Path exportDir = new Path(baseDir, "export");
+      fs.deleteOnExit(exportDir);
+      Path importDirA = new Path(baseDir, "import-a");
+      Path importDirB = new Path(baseDir, "import-b");
+      fs.deleteOnExit(importDirA);
+      fs.deleteOnExit(importDirB);
+      for (Path p : new Path[] {exportDir, importDirA, importDirB}) {
+        assertTrue(fs.mkdirs(p), "Failed to create " + baseDir);
```

Comment:
```suggestion
        assertTrue(fs.mkdirs(p), "Failed to create " + p);
```

Reply:
Nice catch. This is wrong in the test I copied from so I will fix it there as well.